### DISCUSSION TITLE
Everyday Usage

### DISCUSSION THREAD

**Discussion by Jarvvski** [Homebrew/discussions#5623] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.23
ORIGIN: https://github.com/Homebrew/brew
HEAD: 42c11a4dedf352229271593de831161071603c66
Last commit: 3 days ago
Core tap JSON: 20 Sep 12:20 UTC
Core cask tap JSON: 20 Sep 12:20 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Something, somehow, keeps appending

`export PATH="/opt/homebrew/sbin:$PATH"` to my `.zshrc` file

I'm at my ends trying to figure out what is adding it.. I've opened my `.zshrc` sometimes and found lines like;

```
export PATH="/opt/homebrew/sbin:$PATH"
export PATH="/opt/homebrew/sbin:$PATH"
export PATH="/opt/homebrew/sbin:$PATH"
export PATH="/opt/homebrew/sbin:$PATH"
...
```

How.. what.. could be causing this?

**Comment by SMillerDev** [comment]
> How.. what.. could be causing this?

When does it happen? If it happens on login you might have a command that adds in your `.zshrc`

**Reply by Jarvvski** [reply]
Nothing on login, I haven't logged in or out and in the past 24 hours I've seen this:

![CleanShot 2024-09-21 at 12 31 42](https://github.com/user-attachments/assets/f4289eb6-3518-4f94-b0a9-392d8f006cba)
